packages/e2e-tests/jest.performance.config.js

```diff
@@ -1,7 +1,22 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+const { ENVIRONMENT_DIRECTORY = '<rootDir>' } = process.env;
+
 module.exports = {
 	...require( '@wordpress/scripts/config/jest-e2e.config' ),
 	testMatch: [ '**/performance/*.test.js' ],
 	setupFiles: [ '<rootDir>/config/gutenberg-phase.js' ],
+	moduleNameMapper: {
+		// Use different versions of e2e-test-utils for different environments
+		// rather than always using the latest.
```

Comment:
Why?

Reply:
To use the PR's version of `e2e-test-utils` so that changes in that package can be picked up when running the performance tests. This is not needed anymore though since we made the changes opt-in and experimental now. Just forgot to revert it 😅.

Reply:
Unless I'm wrong, this was already the case, PRs use the PR's version of the e2e-test-utils always. There's already a `--tests-env` option in the performance PR?